### PR TITLE
Checking health of the container using SERVER_IP variable

### PR DESCRIPTION
**Pull request by wlp7s0**
SERVER_IP variable will change docker's HEALTHCHECK command if it is set. #471

### CODE REVIEW

**Comment by itzg** [comment]
During testing I found this isn't going to work since defaulting the env variable inside the startup script is too late. Healthchecks now fail with

```
                "Log": [
                    {
                        "Start": "2020-03-31T13:07:26.3312138Z",
                        "End": "2020-03-31T13:07:26.4429726Z",
                        "ExitCode": 1,
                        "Output": "failed to ping --port:25565 : could not connect to Minecraft server: dial tcp: lookup --port: no such host"
                    },
```

Perhaps set the default in the Dockerfile's `ENV` line
https://github.com/itzg/docker-minecraft-server/blob/4ad447ba6c736c250b2478a4dbd884abe6d15f3c/Dockerfile#L68-L74

**Comment by wlp7s0** [comment]
I see, should I remove variable check from `start-finalSetup04ServerProperties` file?

File: start-minecraftFinalSetup [hunk 1]
```diff
@@ -96,6 +96,10 @@ if [ -n "$JVM_DD_OPTS" ]; then
       done
 fi
 
+if [ -z "$SERVER_IP" ]; then
```
Comment (itzg) [inline start-minecraftFinalSetup:99]: ...yes, go ahead and remove this block since it’s too late in the container lifecycle to be used by healthcheck

**Comment by itzg** [comment]
I had to revert this change for now. This was causing all clients to fail to connect via `-p 25565:25565` port forwarding since the minecraft server process was only binding to localhost within the container. Maybe we should introduce a separate `HEALTHCHECK_SERVER_IP` variable.

**Pull request merged**